docs/src/reference/gremlin-variants.asciidoc

```diff
@@ -1369,6 +1369,30 @@ java -cp target/run-examples-shaded.jar examples.BasicGremlin
 java -cp target/run-examples-shaded.jar examples.ModernTraversals
 ----
 
+[[gremlin-java-differences]]
+=== Differences
+
+Gremlin-Java provides additional syntactic sugar that leverages Java's type system. Most notably, the `P.typeOf()` 
+predicate accepts Java `Class` objects directly, providing a more natural and type-safe way to perform type checking:
+
+[source,java]
```

Comment:
any reason the examples in this section are not "live" (i.e. Gremlin Console based)?

Reply:
Updated.